tests/yadism/cf/test_cfh_pol.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import pytest
+from eko.thresholds import ThresholdsAtlas
+
+from yadism import observable_name as obs
+from yadism.coefficient_functions.heavy import f2_nc as h_f2_nc
+from yadism.coefficient_functions.heavy import f3_nc as h_f3_nc
+from yadism.coefficient_functions.heavy import fl_nc as h_fl_nc
+from yadism.coefficient_functions.heavy import g1_nc as h_g1_nc
+from yadism.coefficient_functions.heavy import g4_nc as h_g4_nc
+from yadism.coefficient_functions.heavy import gl_nc as h_gl_nc
+
+
+class MockCouplingConstants:
```

Comment:
This class `MockSF` and `MockESF` can be imported from other test file (`test_nc_asy`)  and avoid repetion

Reply:
This, I am not completely sure. The idea of having different tests is exactly that each one of them should be independent. We surely do not want to run into a cross-dependency of tests in case some of the tests get removed/updated. I think keeping it in the spirit of `test_nc_asy` and `test_nc_kernels` would be best (?).

Reply:
Maybe the best would be to place these functions in `__init__` ?

Reply:
Yes, putting these classes in `__init__`  would be best indeed.

Reply:
let me do it.